synapseclient/models/mixins/__init__.py

```diff
@@ -1,5 +1,9 @@
+"""References to the mixins that are used in the Synapse models."""
+
```

Comment:
Out of scope for this ticket but I think there is also a versionable mixin that currently exists in the client.